crates/lune-roblox/src/instance/mod.rs

```diff
@@ -377,7 +377,7 @@ impl Instance {
             .get_by_ref(self.dom_ref)
             .expect("Failed to find instance in document");
         if let Some(DomValue::Attributes(attributes)) =
-            inst.properties.get(PROPERTY_NAME_ATTRIBUTES)
+            inst.properties.get(&PROPERTY_NAME_ATTRIBUTES.into())
```

Comment:
It might be more readable to do `&ustr(PROPERTY_NAME_ATTRIBUTES)` here, likewise for the other usages, and for usages of `PROPERTY_NAME_TAGS` as well